packages/patternfly-4/react-tokens/src/generateTokens.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable global-require,import/no-dynamic-require,no-restricted-globals */
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Should we remove the no-var-requires rule? Or perhaps have a different linter config for js?

Reply:
Isn't this more of a build script. Should we apply the linter here?